pipeline/mrms_download.py

```diff
@@ -4,16 +4,23 @@
 from urllib.request import urlopen
 
 
-def download(inventory, dest_dir, max_download=4):
-    """Given an inventory file for a particular day, and a destination
-    directory, download all files in the inventory that are not
+def download(inventory, dest_dir = os.path.join(os.getcwd(),'Data'), max_download=4):
+    """Given an inventory file (or dataframe as returned by inventory) 
+    for a particular day, and a destination directory (default to current 
+    directory\Data), download all files in the inventory that are not 
     already in the destination directory with the same size.
 
     While at it, don't download more than `max_download` files in this
     call.
     """
+    
+    try:
+        inv_df = feather.read_feather(inventory)
+    except:
+        feather.write_feather(inventory, 'temp.feather')
```

Comment:
This line confuses me.  I thought that inventory holds a file name and inv_df contains the (possibly empty) data structure. Here, you seem to be writing the file name to `temp.feather`

Reply:
When I first ran the download I tried to pass some non-"feathered" data, This was to check if it was feathered and if not feather it. This just made for a smoother transition so don't have to make sure the feathering is in your script, it will do it for you.